File: app/Fantasy/backend/Models/CoachGameStats.cs
using System.ComponentModel.DataAnnotations;

namespace FantasyApi.Models;

public class CoachGameStats
{
    [Key]
    public Guid Id { get; set; } = Guid.NewGuid();
    public Game? Game { get; set; }
    public Coach? Coach { get; set; }
    public int? CoachTechnicalFouls { get; set; } = 0;
    public int? BenchTechnicalFouls { get; set; } = 0;
    public int? Difference { get; set; }
}

File: app/Statistics/frontend/src/models/CoachGameStats.ts
import type { Coach } from "./Coach";
import type { Game } from "./Game";

export type CoachGameStats = {
  id: string;
  game: Game;
  coach: Coach;
  coachTechnicalFouls: number;
  benchTechnicalFouls: number;
  difference: number;
};
